src/platform/user/profile/constants/addressValidationMessages.js

```diff
@@ -1,31 +1,92 @@
+import React from 'react';
+
+export const ADDRESS_VALIDATION_TYPES = Object.freeze({
+  BAD_UNIT: 'badUnitNumber',
```

Comment:
moved these string into constants as they're being used in three difference places now